collections/Laud_Misc/MS_Laud_Misc_129.xml

```diff
@@ -49,59 +60,482 @@
+                     <locus>(fols. 109v-137v)</locus>
                      <title type="desc" key="work_13047">Lives of Church Fathers</title>
+                     <rubric>De monacho uage mentis ad salutem reducto</rubric>
+                     <incipit>In uno autem monasterii quidam monachus erat, qui adorationem stare non poterat </incipit>
+                     <note>Includes readings on Severus, Martinus, Gregorius, Benedictus, De puerulo liberato, De offerendis, Eulogius, Apollonius, Ammonius, Copres, Mutius, Piamon, Arsenius, Paphnutius, Eulalius. </note>
                      <textLang mainLang="la">Latin</textLang>
                   </msItem>
+                  
+                  <msItem>
+                     <locus>(fols. 138r-149v)</locus>
+                     <title>Theological work</title>
+                     <!--not searched yet if this is an authored work-->
+                     <note>Includes readings on various theological topics:</note>
+                     <rubric>De martirio</rubric>
+                     <incipit>Seruus Dei aduersitate nulla frangitur sed pro ueritatis defensione</incipit>
+                     <rubric>De antichristo</rubric>
+                     <rubric>De die iudici</rubric>
+                     <rubric>De gehenna</rubric>
+                     <rubric>De penis impiorum</rubric>
+                     <rubric>De resurrectione</rubric>
+                     <rubric>De sapientia</rubric>
+                     <rubric>De fide</rubric>
+                     <rubric>De caritate</rubric>
+                     <rubric>De spe</rubric>
+                     <rubric>De gratia</rubric>
+                     <rubric>De conuersis</rubric>
+                     <rubric>De conpunctione</rubric>
+                     <rubric>De confessione</rubric>
+                     <rubric>De manifestis ocultisque peccatis</rubric>
+                     <rubric>De desperatio</rubric>
+                     <rubric>De peccandi consuetudine</rubric>
+                     <rubric>De cogitatione</rubric>
+                     <rubric>De conscientia</rubric>
+                     <rubric>De mendatio</rubric>
+                     <rubric>De iuramento</rubric>
+                     <rubric>De continentia</rubric>
+                     <rubric>De tribulatione iustorum</rubric>
+                     <rubric>De iracundis</rubric>
+                     <rubric>De amicitia munera orta</rubric>
+                     <rubric>De odio</rubric>
+                     <rubric>De tollerantia diuine correptionis</rubric>
+                     <rubric>De temptationibus diaboli</rubric>
+                     <rubric>De inuidia</rubric>
+                     <textLang mainLang="la">Latin</textLang>
+                  </msItem>
+                  
                   <msItem xml:id="MS_Laud_Misc_129-item5" n="5">
+                     <locus>(fols. 149v-158v)</locus>
                      <title type="desc" key="work_10448">Bestiary</title>
```

Comment:
This is the Physiologus Latinus, although there are different versions, and I don't know which one this is

Reply:
yes indeed. It seems to be a previously unknown copy! i'll investigate a bit more. if true, then this could be assigned as 'new discovery' within the project (blog post in making for sure)